use_cases/community-manager.md

```diff
@@ -0,0 +1,220 @@
+    contribution ranking by time period \[year, month, week, day?\].
+    *nominally, I have a concern about these kinds of metrics being
+    \"gameable\", but if they are not visible to contributors
+    themselves, there is less \"gaming\" opportunity.*)
+
+9.  What new contributors submitted their first new patches/issues this
+    week? (*Visualization Note:* New contributors can be colored in
+    visualizations and then additionally a graph can be made for number
+    of)
+
+10. Which contributors became inactive? (Will need a mechanism for
+    setting \"inactive\" thresholds.)
+
+11. Baseline level for the \"average\" repository in an organization and
+    for each, individual organization repository.
+
+12. What projects outside of a community manager's general view (GitHub
+    organization or other boundary) doe my repositories depend on or do
+    my contributors also significantly contribute to?
+
+13. Build a summary report in 140 characters or less. For example,
+    \"Your total commits in this time period \[week? month?\] across the
+    organization increased 12% over the last period. Your most active
+    repositories remained the same. You have 8 new contributors, which
+    is 1 below your mean for the past year. For more information, click
+    here.\"
+
+14. Once a metrics baseline is established, what can be done to move
+    them? [^1]
+
+15. Are there optimal measures for some metrics?
+
+    1.  Pull request size?
+
+    2.  Ratio of maintainers to contributors?
+
+    3.  New contributor to consistent contributor ratio?
+
+    4.  New contributor to maintainer ratio?
+
+
+
+## Metrics
+
+\[Work in progress\]
+
+### Metrics Properties
+
+Right now, based on Augur prototypes and follow up discussions so far,
+we have the following observations that will inform our work both the
+the \"Growth Maturity and Decline\" working group and in Augur
+Development. There are a few things we have learned from prototyping
+Augur with community managers. These features in Augur are particularly
+valued:
+
+1.  Allowing comparisons with projects within a defined universe is
+    essential
+
+2.  Allow community managers to add and remove repositories that they
+    monitor from their repertoires periodically.
+
+3.  Downloadable graphics
+
+4.  Downloadable data (.csv or .json)
+
+5.  Availability of a \"Metrics API\", limiting the amount of software
+    infrastructure the CM needs to maintain for themselves. This is more
+    valued by program managers overseeing larger portfolios right now,
+    but we think has potential to grow as awareness of the relatively
+    light weight of this approach becomes more apparent. By apparent, we
+    really mean \"easy to use and understand\"; right now it is for a
+    programmer, but less so for a community manager without this
+    background or current interest.
+
+### Specific Metrics
+
+1. [Code Commits](../metrics/code-commits.md)
```

Comment:
This should be a link to the "current" merics we're proposing, shouldn't it? If so, it could be Code_Changes.md, I think...